nodes/nodes-nodes-modifying.adoc

```diff
@@ -0,0 +1,40 @@
+// For example:
+// * file name: assembly_my-assembly-a.adoc
+// * ID: [id='assembly_my-assembly-a_{context}']
+// * Title: = My assembly A
+//
+// The ID is used as an anchor for linking to the module.
+// Avoid changing it after the module has been published
+// to ensure existing links are not broken.
+//
+// In order for  the assembly to be reusable in other assemblies in a guide,
+// include {context} in the ID: [id='a-collection-of-modules_{context}'].
+//
+// If the assembly covers a task, start the title with a verb in the gerund
+// form, such as Creating or Configuring.
+// The `context` attribute enables module reuse. Every module's ID
+// includes {context}, which ensures that the module has a unique ID even if
+// it is reused multiple times in a guide.
+:context: nodes-nodes-modifying
+[id='nodes-nodes-modifying_{context}']
+= Modifying existing nodes in your {product-title} cluster
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+
+As an administrator, you can use the node group configuration map or through the use of labels on the nodes. 
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+include::modules/nodes-nodes-modifying-about.adoc[leveloffset=+1]
+
+include::modules/nodes-nodes-modifying-resources.adoc[leveloffset=+1]
+
+include::modules/nodes-nodes-modifying-marking.adoc[leveloffset=+1]
```

Comment:
this should be in, and is somewhat redundant with, content in the "Working with nodes" story

Reply:
Moved this file.